js/src/pages/price-benchmark/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { getQuery } from '@woocommerce/navigation';
+import { Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import MainTabNav from '~/components/main-tab-nav';
+import IssueTypeNavigation from './table-type-navigation';
```

Comment:
@ankitrox Should the component be `TableTypeNavigation` instead of `IssueTypeNavigation`?
```suggestion
import TableTypeNavigation from './table-type-navigation';
```

Reply:
Updated this.